.github/workflows/triage_handle_new_comments.yml

```
@@ -0,0 +1,33 @@
+name: 'Handle Issue/PR Comment Workflow'
+on:
+  # makes this workflow reusable
+  workflow_call:
+    secrets:
+      ADD_TO_TRIAGE_BOARD_TOKEN:
```

Should we just change this name now?
```suggestion
      TRIAGE_BOARD_TOKEN:
```

---

I'd rather not since I would have to have an org owner create a new one and update all the repos that use that org level secret

---

We can leave the existing one there so everything doesn't need to be updated at once but I think we should use a more generic named token for all new existing stuff.